sdk/python/feast/infra/offline_stores/offline_store.py

```diff
@@ -179,6 +179,19 @@ def pull_latest_from_table_or_query(
         end_date: datetime,
     ) -> RetrievalJob:
         """
+        This method is invoked when running materialization (using the `feast materialize` or `feast
```

Comment:
super nitty: can you put `This method pulls data from the offline store, and the FeatureStore class is used to write this data into the online store.` before `This method is invoked...`? in general I think a docstring should first describe what the function does, and only afterwards provide details on how it should be used, etc.